### PR TITLE
Add Hessenschau

### PR DESCRIPTION
**Pull request by martinknz**

### CODE REVIEW

File: src/fundus/publishers/de/__init__.py [hunk 1]
```diff
@@ -240,3 +241,13 @@ class DE(PublisherEnum):
         ],
         parser=RheinischePostParser,
     )
+
+    Hessenschau = PublisherSpec(
+        name="Hessenschau",
+        domain="https://www.hessenschau.de/",
+        sources=[
+            RSSFeed("https://www.hessenschau.de/index.rss"),
+            Sitemap("https://www.hessenschau.de/indexsitemap.nc.xml"),
```
Comment (MaxDall) [inline src/fundus/publishers/de/__init__.py:250]:
```python
            RSSFeed("https://www.hessenschau.de/index.rss"),
            NewsMap("https://www.hessenschau.de/newssitemap.nc.xml"),
            Sitemap("https://www.hessenschau.de/sitemap.nc.xml"),
```

File: src/fundus/publishers/de/hessenschau.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_date_parsing,
+)
+
+
+class HessenschauParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("p.copytext__text.text__copytext")
```
Comment (MaxDall) [inline src/fundus/publishers/de/hessenschau.py:15]: It seems that the first paragraph is often but not always the article summary.

File: src/fundus/publishers/de/hessenschau.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                subheadline_selector=self._subheadline_selector,
+            )
+
+        @attribute
+        def title(self) -> Optional[str]:
+            return self.precomputed.meta.get("og:title")
+
+        @attribute
+        def publishing_date(self) -> Optional[datetime]:
+            return generic_date_parsing(self.precomputed.meta.get("article:published_time"))
```
Comment (MaxDall) [inline src/fundus/publishers/de/hessenschau.py:32]: Could you add `authors` and `topics` as well? You can find both of them in the meta tags. Just refer to other parsers to see how to extract them.

**Review by MaxDall — CHANGES_REQUESTED** [review]
Thanks for adding 👍

**Comment by MaxDall** [comment]
@martinknz I fixed both selectors for you. This should be good to go now.

**Review by MaxDall — APPROVED** [review]

**Review by addie9800 — APPROVED** [review]

**Pull request merged**